### ISSUE TITLE
On multiple windows uses another project's .stylelintrc

### ISSUE THREAD

**Issue by geminorum** [AtomLinter/linter-stylelint#376]
When I open multiple projects the linter-stylelint uses one of project's `.stylelintrc` for linting on all others; Although the others have no config files and the `Disable when no config file is found` is checked.

**Comment by emuvente** [comment]
@geminorum I suspect that this issue may be fixed with version 4.3.0 of `linter-stylelint`. Are you still having the same problem?

**Comment by geminorum** [comment]
nope. still having the problem on linter-stylelint 4.3.0 and atom 1.26.1
